iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/TokenRenewalTests.java

```diff
@@ -102,7 +102,9 @@ public static void stopProxy()
      * @throws Exception if the test fails in any way
      */
     @Test
-    @ContinuousIntegrationTest
+    // While this test is long enough to be considered a continuous integration test, it is the only test that truly validates
+    // that the device client's token renewal logic works. Because of that, it needs to be run at the gate.
+    //@ContinuousIntegrationTest
     public void tokenRenewalWorks() throws Exception
```

Comment:
This is a 15 minute test, just FYI. I don't love putting this in the gate, but since this regression slipped by in part because this test isn't at the gate, it feels wise to add it back to the gate